### ISSUE TITLE
Feature for COCO on-line Test images

### ISSUE THREAD

**Issue by Jian-Xi** [aimagelab/meshed-memory-transformer#20]
Thanks for sharing the code of this brilliant work!

I'm wondering that is it possible to make the detection file for coco online test images available as the one for train/val hdf5 file. Or is there some available online resource that I did not spot it.

Thanks in advance!

**Comment by marcellacornia** [comment]
Hi @Jian-Xi,
thanks for your interest in our work.

I just uploaded the detection feature file for the COCO only test set at this [Google Drive link](https://drive.google.com/file/d/1n9Cx6TgaQnx4rP0UoiXUmS75RWDOtJjF/view?usp=sharing).

Let me know if you can download it properly.

**Comment by dkm17113034** [comment]
Thanks for sharing the google drive link, but after downloading using ubuntu terminal it's showing 3kb 
file. What's is wrong with this?

**Comment by dkm17113034** [comment]
it's downloading HTML document not hdf5 file

**Comment by dkm17113034** [comment]
It's saying that too many users have downloaded or viewed this file. It's not downloading.

**Comment by Jian-Xi** [comment]
Hey @marcellacornia, thanks for your response.
The upload works. I downloaded the feature file successfully. I'd like to have a additional question about this file. 
As it states in all papers that there are 40778 images in online test dataset. The extracted feature file however, includes features for 81239 images. I'm a little confused now about the data file. What's the difference here.
Any further inputs will be appreciated. Thanks again.

**Comment by HN246** [comment]
@Jian-Xi, have you downloaded the file successfully??? I can not open the file now. Can you share your file on a new path or on Baidu Netdisk??  Much thanks for you!!!

**Comment by Jian-Xi** [comment]
Hey @HN246,

Yes, I did download it successfully. Yet I can not provide it via Baidu Netdisk since I do not have an account for this net disk.
You can prefer to check this repository out instead [https://github.com/MILVLG/bottom-up-attention.pytorch](bottom-up-attention.pytorch). The extracted features are quite similar, maybe a little biased but it works (this reflects on the captioning results, some of them captioning sentences are not complete, yet the content is coherent). What you additionally need to do is to save the extracted features into the same format as @marcellacornia  does if you want to use this model. 

Hope it helps you somehow. 
Cheers
